ir/src/main/scala/org/scalajs/ir/Serializers.scala

```diff
@@ -1003,21 +1007,24 @@ object Serializers {
     }
 
     def readTree(): Tree = {
+      val tag = readByte()
       val pos = readPosition()
```

Comment:
Have you considered moving `readPosition()` into `readTreeFromTag`? That would simplify the call-sites.

Reply:
Good point. Done.